project/src/main/java/com/google/autograder/data/Database.java

```diff
@@ -139,6 +147,44 @@ public String getAllQuestionsJSON(String key) {
     return new Gson().toJson("");
   }
 
+/** 
+ * this function takes the assignment Key and answer group key as parameters
+ * returns list of blobKeys to submissions that fit in specified answer group for specific question
+ */ 
+  public String blobkeysFromAnswerGroup(String assignKey, String groupKey) {
```

Comment:
Would it be better if returning a List<String> instead? (More consistent with naming and comment)

Where is this used?

Reply:
I originally had it return a List but Mahima suggested returning a JSON so that when displaying the submission groupings, the user would be able to see which student's submission is which.

Reply:
I think other methods in this class, if they return a JSON string, have names that end in "JSON"

Reply:
ok